### ISSUE TITLE
Error for a specific file

### ISSUE THREAD

**Issue by davidanthoff** [felipenoris/XLSX.jl#104]
I tried to read [this](https://github.com/anthofflab/MimiRICE2010.jl/blob/master/data/RICE_2010_base_000.xlsm), but I get an error message saying:
```julia
ERROR: LoadError: AssertionError: !(isopen(state))
Stacktrace:
 [1] iterate(::XLSX.SheetRowStreamIterator, ::XLSX.SheetRowStreamIteratorState) at C:\Users\david\.julia\packages\XLSX\2GLh4\src\stream.jl:122
 [2] iterate(::XLSX.WorksheetCache{XLSX.SheetRowStreamIterator}, ::Int64) at C:\Users\david\.julia\packages\XLSX\2GLh4\src\stream.jl:238
 [3] open_or_read_xlsx(::String, ::Bool, ::Bool, ::Bool) at C:\Users\david\.julia\packages\XLSX\2GLh4\src\read.jl:230
 [4] readxlsx at C:\Users\david\.julia\packages\XLSX\2GLh4\src\read.jl:37 [inlined]
 [5] getrice2010parameters(::String) at C:\Users\david\.julia\dev\MimiRICE2010\src\parameters.jl:9
 [6] #getrice#3 at C:\Users\david\.julia\dev\MimiRICE2010\src\MimiRICE2010.jl:160 [inlined]
 [7] getrice() at C:\Users\david\.julia\dev\MimiRICE2010\src\MimiRICE2010.jl:160
 [8] top-level scope at none:0
 [9] include at .\boot.jl:326 [inlined]
 [10] include_relative(::Module, ::String) at .\loading.jl:1038
 [11] include(::Module, ::String) at .\sysimg.jl:29
 [12] include(::String) at .\client.jl:403
 [13] top-level scope at none:0
in expression starting at C:\Users\david\.julia\dev\MimiRICE2010\test\runtests.jl:8
ERROR: Package MimiRICE2010 errored during testing
```
This is running inside a larger test suite, but I can replicate the same behavior by just trying to read the file by itself.

**Comment by felipenoris** [comment]
omg... not the first time that the Rice workbook pushes XLSX to its limit. 😄 . (https://github.com/felipenoris/XLSX.jl/issues/18)

**Comment by felipenoris** [comment]
I was able to replicate the issue.

**Comment by davidanthoff** [comment]
Haha :) We actually didn't create it, its Bill Nordhaus' work.

**Comment by ronisbr** [comment]
I am trying to read/write a file with some charts and I am having the same problem. Is there any workaround?

**Comment by felipenoris** [comment]
@ronisbr , can you provide a minimal Excel file that reproduces the bug? I would like to add it to this repo for tests.

**Comment by jd-foster** [comment]
Regarding the original issue, I have created two files derived from the RICE workbook.

The first file, [RICE_min_Error.xlsx](https://github.com/felipenoris/XLSX.jl/files/3455224/RICE_min_Error.xlsx), has just two of the sheets, Controls and Data, and produces errors.
The error is connected to the sheet Data.

This issue seems to be related to the identification in XLSX.jl of the range of the sheet Data, in this case "A1:DR455", which is different to the "natural" range you might expect in Excel.  However, the Control sheet also exhibits this type of thing with a range of "A1:BS98" while the "natural" range is "A1:BI90".

The second file [RICE_min_No_error.xlsx](https://github.com/felipenoris/XLSX.jl/files/3455232/RICE_min_No_error.xlsx) is derived from the first by deleting all the data from the Data sheet. It is read in without error.

**Comment by AdilsonAtuario** [comment]
Ei Felipe, você é brasilsileiro, certo? Não to acreditando nisso. Mas quando vi seu sobrenome, já desconfiei. kkk. Seu pacote XLSX é sensacional!!! tá salvando muito minha vida, cara.
descupe aí pelo português num forum em inglês. vou tentar escrever inglês nos próximos.
Mas to muito feliz e orgulhoso de ver um brasileiro neste projeto. A programação Julia é muuuito bom.

**Comment by felipenoris** [comment]
Fixed by 0b13cf5dfebe4ed9eda85b6e11bf4e6a2be6679a

**Comment by davidanthoff** [comment]
Great, thanks!
